weblate/trans/forms.py

```diff
@@ -2380,3 +2388,23 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.helper = FormHelper(self)
         self.helper.form_tag = False
+
+
+class ProjectTokenDeleteForm(forms.Form):
+    token = forms.CharField(required=True, widget=forms.HiddenInput)
```

Comment:
Perhaps use `forms.IntegerField` instead? Even better would be `forms.ModelChoiceField`, the form then returns directly the object, and you can skip additional validation in the view.

Reply:
Changed to IntegerField - I had ModelChoiceField but this did not seem to work.